WooCommerce/Classes/ViewModels/Order Details/OrderPaymentDetailsViewModel.swift

```diff
@@ -111,6 +109,19 @@ final class OrderPaymentDetailsViewModel {
         return String.localizedStringWithFormat(template, styleDate, order.paymentMethodTitle)
     }
 
+    /// Awaiting payment
+    ///
+    var awaitingPayment: String? {
+        if order.paymentMethodTitle.isEmpty {
+            return String.localizedStringWithFormat(
+                NSLocalizedString("Awaiting payment", comment: "Awaiting payment"))
+        }
+        return String.localizedStringWithFormat(
+            NSLocalizedString("Awaiting payment via %@",
+                              comment: "Awaiting payment via (payment method title)"),
```

Comment:
Nit: same comment as above

```suggestion
            NSLocalizedString("Awaiting payment via %@",
                              comment: "The title on the payment row of Order Detail screen when the payment for a specific payment method is still pending. Reads like: Awaiting payment via Stripe."),
```